### PR TITLE
BLADE-481 Ignore BridJ intermittent erroneous failure

### PR DESCRIPTION
**Pull request by christopherbryanboyd**
Sent from https://github.com/christopherbryanboyd/liferay-blade-cli/pull/36

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: cli/src/test/java/com/liferay/blade/cli/BladeTest.java [hunk 1]
```diff
@@ -89,27 +89,34 @@ public void run(String[] args) throws Exception {
 
 				StringBuilder sb = new StringBuilder();
 
+				boolean bridj = false;
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/BladeTest.java]:
```suggestion
				boolean bridjError = false;
```

File: cli/src/test/java/com/liferay/blade/cli/TestUtil.java [hunk 2]
```diff
@@ -63,27 +63,34 @@ public static BladeTestResults runBlade(
...
+
 		try (Scanner scanner = new Scanner(error)) {
-			while (scanner.hasNextLine()) {
+			while (scanner.hasNextLine() && !bridj) {
 				String line = scanner.nextLine();
 
 				if ((line != null) && (line.length() > 0)) {
-					if (line.startsWith("SLF4J:")) {
-						continue;
+					if (line.contains("org/bridj/Platform$DeleteFiles")) {
```
Comment (gamerson) [inline cli/src/test/java/com/liferay/blade/cli/TestUtil.java]: Does this code need to be in both TestUtil and in BladeTest?  Can't we just have this in one place?

**Comment by christopherbryanboyd** [comment]
ci:test

**Comment by christopherbryanboyd** [comment]
ci:retest

**Comment by christopherbryanboyd** [comment]
ci:retest

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**